tests/e2e-udx/udx-cpp/setup-vdb/base/setup-vdb.yaml

```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: vertica.com/v1beta1
+apiVersion: vertica.com/v1
```

Comment:
How exactly can v1 and v1beta1 coexist? You moved some function from v1beta1/verticadb_types to v1/verticadb_types and you use the same images for e2e tests.

Reply:
The controller only works with v1. That's why some of the functions in v1beta1 could be moved out. There is a conversion function, which is called by a webhook to easily move between v1beta1 and v1 versions. If you are up for reading a bit I found this tutorial went over it quite well: https://book.kubebuilder.io/multiversion-tutorial/tutorial. I can also do a code walk-through.